src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,120 @@
+   * when `isInvalid` is true. */
+  invalidText?: string;
+  /** When using the Toggle as a "controlled" form element, you can specify
+   * the Toggle's checked state using this prop.
+   * Learn more about controlled and uncontrolled form fields:
+   * https://goshakkk.name/controlled-vs-uncontrolled-inputs-react/ */
+  isChecked?: boolean;
+  /** Adds the 'disabled' and `aria-disabled` attributes to the input when true.
+   * This also makes the text italic and color scheme gray. */
+  isDisabled?: boolean;
+  /** Adds the 'aria-invalid' attribute to the input when true. This also makes
+   * the color theme "NYPL error" red for the button and text. */
+  isInvalid?: boolean;
+  /** Adds the 'required' attribute to the input when true. */
+  isRequired?: boolean;
+  /** The checkbox's label. This will serve as the text content for a `<label>`
+   * element if `showlabel` is true, or an "aria-label" if `showLabel` is false. */
+  labelText: string;
+  /** The name prop indicates into which group of checkboxes this checkbox
+   * belongs. If none is specified, 'default' will be used */
+  name?: string;
+  /** The action to perform on the `<input>`'s onChange function  */
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  /** Offers the ability to hide the helper/invalid text. */
+  size?: ToggleSizes;
+}
+
+export const onChangeDefault = () => {
+  return;
+};
+
+/**
+ * Component that renders Chakra's `Switch` component along with NYPL defaults.
+ */
+const Toggle = React.forwardRef<HTMLInputElement, ToggleProps>(
+  (props, ref?) => {
+    const {
+      additionalStyles = {},
+      invalidText,
+      defaultChecked = false,
+      helperText,
+      id = generateUUID(),
+      isChecked,
+      isDisabled = false,
+      isInvalid = false,
+      isRequired = false,
+      labelText,
+      name,
+      onChange = onChangeDefault,
+      size = ToggleSizes.Large,
+    } = props;
+    const footnote = isInvalid ? invalidText : helperText;
+    const ariaAttributes = {};
+    const styles = useMultiStyleConfig("Toggle", {});
+    const switchStyles = useStyleConfig("Switch");
+    ariaAttributes["aria-label"] =
+      labelText && footnote ? `${labelText} - ${footnote}` : labelText;
+
+    return (
+      <>
+        <Box display="flex" __css={{ ...styles, ...additionalStyles }}>
+          <Switch
+            id={id}
+            name={name || "default"}
+            isDisabled={isDisabled}
+            isInvalid={isInvalid}
+            isRequired={isRequired}
+            ref={ref}
+            size={size}
+            {...(isChecked !== undefined
+              ? {
+                  isChecked,
+                  onChange,
+                }
+              : {
+                  defaultChecked,
+                })}
+            {...ariaAttributes}
+            __css={switchStyles}
+          />
+          <Box __css={styles.label}>
+            <Label htmlFor={id}>{labelText}</Label>
+          </Box>
```

Comment:
You don't need this. Chakra's `Switch` component is a label wrapping an input. So you can do:
```
...
  __css={switchStyles}
>
  {labelText}
</Switch>
```

Right now, there are two labels rendering so we should make use of the existing one coming from Chakra.

Reply:
So looking there's something strange going on here.  When I just do the suggestion above, the text appears outside the `label` element. 
![Screen Shot 2021-12-15 at 3 16 24 PM](https://user-images.githubusercontent.com/3579472/146259269-6e231e81-34d8-47cf-9035-87cb3a135635.png)

Looking at the `switch` documentation, it looks like they have two `label` elements - https://chakra-ui.com/docs/form/switch#usage

Does this look like a bug in Chakra?  I'm happy to send them a PR if that's the case
![Screen Shot 2021-12-15 at 3 21 54 PM](https://user-images.githubusercontent.com/3579472/146259489-77c7cb35-27f8-4541-b1c6-089f4abb58f6.png)